src/monaco-converter.ts

```diff
@@ -477,30 +477,33 @@ export class ProtocolToMonacoConverter {
         };
     }
 
-    asDocumentSymbolResult(values: SymbolInformation[] | DocumentSymbol[]): monaco.languages.SymbolInformation[] {
+    asDocumentSymbolResult(values: SymbolInformation[] | DocumentSymbol[]): monaco.languages.DocumentSymbol[] {
         if (DocumentSymbol.is(values[0])) {
             // FIXME when Monaco supports DocumentSymbol
```

Comment:
@Twinside this is great! Would you be able also to add missing conversion? Now it should be possible. We should do it by analogy with https://github.com/Microsoft/vscode-languageserver-node/blob/7850be5cce4ed39485150969de14947d08c47375/client/src/protocolConverter.ts#L517-L542